core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -450,9 +450,11 @@ export class Topic {
       subtopicId: number, fromIndex: number, toIndex: number): void {
     const subtopic = this.getSubtopicById(subtopicId);
     if (subtopic !== null) {
-      const skillToMove = cloneDeep(subtopic.getSkillSummaries()[fromIndex]);
-      subtopic._skillSummaries.splice(fromIndex, 1);
-      subtopic._skillSummaries.splice(toIndex, 0, skillToMove);
+      const skillToMove = cloneDeep(subtopic?.getSkillSummaries()[fromIndex]);
+      subtopic?._skillSummaries.splice(fromIndex, 1);
```

Comment:
Why is the `?` needed? I think TS checker should be able to understand that subtopic is not null.